engines/bladerunner/bladerunner.h

```diff
@@ -351,10 +351,10 @@ static inline void drawPixel(Graphics::Surface &surface, void* dst, uint32 value
 			*(uint8*)dst = (uint8)value;
 			break;
 		case 2:
-			*(uint16*)dst = (uint16)value;
```

Comment:
are these really needed? dst should be always aligned properly for each pixel format and selected write, it read subsequently by cpu or gpu